test_framework/scripts/longhorn-setup.sh

```diff
@@ -33,7 +33,7 @@ check_longhorn_status() {
 
 install_csi_snapshotter_crds(){                                                 
     git clone --single-branch --branch "master" "https://github.com/kubernetes-csi/external-snapshotter.git" /tmp/k8s-csi-external-snapshotter
-    kubectl apply -f /tmp/k8s-csi-external-snapshotter/client/config/crd \      
```

Comment:
@yasker extra spaces here caused the issue, and i decided to switch to bash since we had some other syntax compatibility issues before between `bash` and `ash`